### DISCUSSION TITLE
v1.2.0-beta.3 创建的资源没有ownerReferences了吗？

### DISCUSSION THREAD

**Discussion by shikingram** [kubevela/kubevela#2999] [Q&A] [answered]
如题，下午刚升级的，之前是`1.1.9`还看到是`$applicationName-$version-$namespace`的格式，请问下是否删除了？

**Comment by wonderflow** [comment, marked as answer]
对。去掉了。

Yes, we removed ownerReferences on resources, as KubeVela is a control plane with multiple clusters to manage. Resources that have ownerRef can't be managed efficiently.
